tests/preprocessing/test_util.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+import pytest
+
+from trackintel.preprocessing.util import calc_temp_overlap
+
+
+@pytest.fixture
+def time_1():
```

Comment:
I would say that the fixtures here make a lot of sense even though they are as detailed as in the other test file.

Reply:
Yes they are reused